client/verta/verta/registry/_verta_model_base.py

```diff
@@ -67,7 +67,7 @@ def predict(self, input):
             will be fully compatible with the Verta platform as you iterate
             locally.
 
-            :meth:`predict` must be written to both recieve [1]_ and return
+            :meth:`predict` must be written to both receive [1]_ and return
```

Comment:
UGH gets me every time

Reply:
I of course can't remember it now but I have a similar nemesis